experimental/devices/unicornhd/unicornhd.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package unicornhd implements interfacing code to Pimoroni's Unicorn HD hat.
+package unicornhd
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	"image/draw"
+
+	"periph.io/x/periph/conn/spi"
+)
+
+const (
+	Height = 16
+	Width  = 16
+	speed  = 9000000
+	bits   = 8
+	prefix = 0x72
+)
+
+// Dev represents a Unicorn HAT HD (https://shop.pimoroni.com/products/unicorn-hat-hd)
+// connected over a SPI port.
+type Dev struct {
+	// Communication
+	connector spi.Conn
+	pixels    *image.NRGBA
+	txBuffer  []byte
+	err       error
+}
+
+// New returns a unicornHD driver that communicates over SPI.
+//
+// The SPI port speed must be 9MHz and the SPI mode, 0, as in the
+// python example library.
+func NewUnicornhd(port spi.Port) (*Dev, error) {
+	connector, err := port.Connect(speed, spi.Mode0, bits)
```

Comment:
Inline speed, don't make it a const. Please switch to the new format: 9 * physic.MegaHertz.
Inline bits.